packages/grafana-o11y-ds-frontend/tsconfig.json

```diff
@@ -5,9 +5,15 @@
     "declarationDir": "./compiled",
     "emitDeclarationOnly": true,
     "isolatedModules": true,
+    "allowJs": true,
```

Comment:
Is this needed for this dep bump? 🤔

Reply:
naw it's needed to be able to include `setupTests`. that file imports a `.js` script, and without setting `allowJs: true` there are type errors